packages/zoe/src/contracts/loan/liquidate.js

```diff
@@ -54,9 +57,24 @@ export const doLiquidation = async (
       Loan: loanPayout,
     });
     await depositToSeat(zcf, lenderSeat, amounts, payoutPayments);
-    lenderSeat.exit();
-    collateralSeat.exit();
-    zcf.shutdown('your loan had to be liquidated');
+
+    const closeSuccessfully = () => {
+      console.log('close successful');
```

Comment:
Did you mean to drop the console.log() lines (here and 69) or are they intentional?

Reply:
Oh, I mean to drop these